x-pack/plugins/apm/public/components/app/TransactionDetails/Transaction/WaterfallContainer/index.tsx

```diff
@@ -4,10 +4,10 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import { Location } from 'history';
 import React from 'react';
-import { Transaction } from '../../../../../../typings/es_schemas/Transaction';
-
 import { IUrlParams } from 'x-pack/plugins/apm/public/store/urlParams';
+import { Transaction } from '../../../../../../typings/es_schemas/Transaction';
```

Comment:
when should we use relative vs `x-pack/..` import paths? it seems like we're inconsistent in some places

Reply:
Good point. I just let vscode decide (auto-importing). As a rule of thumb, if it gets above four `../` deep it should probably use the absolute import notation.
I'll change it manually here.